config/routes.rb

```diff
@@ -2,11 +2,13 @@
 
 Rails.application.routes.draw do
   root 'main#home'
+  get '/home', to: 'main#homeregistered', as: 'home'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.